lib/beaker/subcommand.rb

```diff
@@ -49,55 +53,134 @@ def initialize(*args)
     class_option :'exclude-tags', :type => :string, :group => 'Beaker run'
     class_option :'xml-time-order', :type => :boolean, :group => 'Beaker run'
 
-    desc "init HYPERVISOR", "Initialises the beaker test environment configuration"
+    desc "init BEAKER_RUN_OPTIONS", "Initializes the required configuration for Beaker subcommand execution"
     option :help, :type => :boolean, :hide => true
     long_desc <<-LONGDESC
+      Initializes the required .beaker configuration folder. This folder contains
+      a subcommand_options.yaml file that is user-facing; altering this file will
+      alter the options subcommand execution.
 
-      Initialises a test environment configuration for a specific hypervisor
-
-      Supported hypervisors are: vagrant (default), vmpooler
-
+      Also modified in this operation is the .subcommand_state file that is used
+      by beaker to determine subcommand state. You as a user should never have to
+      edit this file.
     LONGDESC
```

Comment:
I modified that in the previous PR to not expose any content about what's happening with that .subcommand_state.  I also added some more messaging about the subcommand_options.yaml file location and saving to it.